addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/RFXComBindingConstants.java

```diff
@@ -64,6 +64,8 @@
             BRIDGE_RFXTRX315, BRIDGE_RFXREC443);
 
     // List of all Channel ids
+    public final static String CHANNEL_RAW_MESSAGE = "rawMessage";
+    public final static String CHANNEL_RAW_PAYLOAD = "rawPayload";
```

Comment:
I refer to capital M/P there are some (closed) pull requests for the rain-message because there was something wrong with the caps there.

Here you have  _rawMessage_ in the xml you have _rawmessage_